realm/realm-library/src/main/cpp/io_realm_internal_objectstore_OsApp.cpp

```diff
@@ -57,7 +57,7 @@ struct AndroidClientListener : public realm::BindingCallbackThreadObserver {
         JniUtils::detach_current_thread();
     }
 
-    void handle_error(std::exception const& e) override
+    bool handle_error(std::exception const& e) override
```

Comment:
`BindingCallbackThreadObserver::handle_error` signature now requires the return of a boolean. Because our logic always handles the exception we always return true.